docs/skeleton_screens.md

```diff
@@ -0,0 +1,44 @@
+# Skeleton screens
+
+In order to improve both UX and the performance score
+you can choose to use skeleton screens
+in situation where you need to fill the interface
+with data from a requests to an external service.
+
+## Main Purpose
+
+The skeleton screens are being showed instantly
+in order to deliver some content to the end user
+fast while loading data.
+When the data is arriving the skeleton screens
+are being replaced with the real data.
+
+## How to use it
+
+The skeleton screens are rendered with help from the
+[skeleton-screen-css library](https://www.npmjs.com/package/skeleton-screen-css?activeTab=readme).
+ By using [ssc classes](https://github.com/nullilac/skeleton-screen-css#example---card)
+ you can easily compose screens
+ that simulate the look of a "real" rendering with real data.
+
+### Simple example
+
+In this example we have a HelloWorld app with three text props attached:
+
+```tsx
+import React from "react";
+
+const SearchResultListItemSkeleton: React.FC = () => {
+  return (
+    <article className="search-result-item ssc">
+      <div className="ssc-square">&nbsp;</div>
+      <div className="ssc-wrapper w-80">
+        <div className="ssc-head-line w-60 mb" />
```

Comment:
Can we provide any help, hints or guidelines about how developers should use standard classes like `.search-result-item` and skeleton classes?

One example might be: "Use BEM block classes (for layout in relation to other elements etc.) but use SSC classes instead of elements." Note: I do not know if this example will work in practice.

Reply:
Done